mozci/util/taskcluster.py

```diff
@@ -124,3 +125,29 @@ def get_taskcluster_options():
         options["rootUrl"] = "https://community-tc.services.mozilla.com"
 
     return options
+
+
+def send_admin_emails(subject, content, emails):
```

Comment:
Why 'admin'?

Reply:
Because I thought we would only have one list of emails in the secret that would be the admins